### PR TITLE
Movie details/about movie media

### PR DESCRIPTION
**Pull request by PawelOwiesek**
MediaQeries for movieDetails, all conflicts solved.

### CODE REVIEW

File: src/features/MovieDetails/styled.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ export const Title = styled.h1`
 `;
 export const MainInfo = styled.div`
   display: grid;
-  grid-template-rows: repeat(3 auto);
+  grid-template-rows: repeat(2, auto);
   position: absolute;
   bottom: 56px;
   left: 276;
```
Comment (MirekSobota) [inline src/features/MovieDetails/styled.js:168]: You need to add `px` here i think.

**Review by MirekSobota — APPROVED** [review]
Good job, a lot of css :)

**Review by DorotaZielinska — APPROVED** [review]

File: src/features/MovieDetails/styled.js (hunk 1 above)
Comment (PawelOwiesek) [reply src/features/MovieDetails/styled.js:168]: Its repeat 2 x auto

**Comment by DorotaZielinska** [comment]
Great work.

**Pull request merged**